src/Infrastructure/Stize.Infrastructure.Azure/Sql/SqlDatabaseExtensions.cs

```diff
@@ -67,36 +66,140 @@ public static SqlDatabaseBuilder ElasticPoolId(this SqlDatabaseBuilder builder,
         }
 
         /// <summary>
-        //     The edition/tier of the database to be created. Applies only if `create_mode` is `Default`.
-        //     Valid values are: `Basic`, `Standard`, `Premium`, `DataWarehouse`, `Business`,
-        //     `BusinessCritical`, `Free`, `GeneralPurpose`, `Hyperscale`, `Premium`, `PremiumRS`,
-        //     `Standard`, `Stretch`, `System`, `System2`, or `Web`. Please see [Azure SQL Database
-        //     Service Tiers](https://azure.microsoft.com/en-gb/documentation/articles/sql-database-service-tiers/).
+        /// Restores a deleted database using the Restorable Dropped Database Resource ID.
         /// </summary>
         /// <param name="builder"></param>
-        /// <param name="edition"></param>
+        /// <param name="databaseId">Resource ID of the Restorable Dropped Database to restore from</param>
+        /// <param name="pointInTime">Point in time that you want to restore from. Only specify to restore from an earlier point in time.</param>
         /// <returns></returns>
-        public static SqlDatabaseBuilder SkuTier(this SqlDatabaseBuilder builder, Input<string> edition)
+        public static SqlDatabaseBuilder SetAsRestore(this SqlDatabaseBuilder builder, Input<string> databaseId, Input<string> pointInTime = null)
```

Comment:
Should we rename this to "CreateAsRestoreOf"? I feel the current name is a bit confusing.